### DISCUSSION TITLE
Sidebar Item Label and Header of the Page

### DISCUSSION THREAD

**Discussion by ganeshkanthe** [GitbookIO/community#579] [1. How to]
Currently, the sidebar item label (Let us say, **Onboarding Corporates**) is appearing as a header of the corresponding item page (as **Onboarding Corporates**). I want the sidebar item label different than its corresponding page header. Is it possible to configure in GitBook or through my GitHub codebase (which I sync with GitBook)?
![sample](https://github.com/GitbookIO/community/assets/163671345/7783c448-c3fb-4995-8992-e3c856d37ccc)

I have attached a file FYR. Please help.

**Comment by aikoii** [comment]
👋 @ganeshkanthe, GitBook automatically syncs the page title as the 'sidebar label', which also applies to content synced from GitHub. This means the 'sidebar label' and page title are the same by default, and currently, there's no option to change this.
